enos/enos-scenario-upgrade.hcl

```diff
@@ -210,10 +249,29 @@ scenario "upgrade" {
     }
   }
 
+  step "get_updated_vault_get_cluster_ips" {
```

Comment:
something like "get_updated_cluster_ips" might be better?

Reply:
updated to `get_vault_cluster_ips` and `get_updated_vault_cluster_ips`